docs/app/Examples/elements/List/Content/ListHeaderExample.js

```diff
@@ -8,6 +8,16 @@ export default class ListHeaderExample extends Component {
         <List.Item header='Chapter 1' description='The chapter in which we meet the characters' />
         <List.Item header='Chapter 2' description='The chapter in which the bad guy is introduced' />
         <List.Item header='Chapter 3' description='Spoiler alert: The chapter in which the good guy wins!' />
+        <List.Item>
+          <List.ItemContent>
```

Comment:
Correct, no content when there is no icon/image. Reference the SUI docs for the exact markup. 

http://semantic-ui.com/elements/list

In the case of the Item view, adding content with no icon/image will actually break the vertically aligned variations.